android/app/src/main/java/org/fossasia/openevent/activities/LocationActivity.java

```diff
@@ -101,7 +117,38 @@ protected void onResume() {
         realmRepo.getSessionsByLocation(location)
                 .addChangeListener((sessions, orderedCollectionChangeSet) -> {
                     mSessions.clear();
+                    String upcomingTitle = "";
+                    String track = "";
+                    String color = null;
```

Comment:
When using local variables, write them in one line. Codacy issue for per line variables should be for global variables only IMO